owncloudApp/src/main/java/com/owncloud/android/workers/UploadFileFromFileSystemWorker.kt

```diff
@@ -250,8 +250,8 @@ class UploadFileFromFileSystemWorker(
 
         val result = executeRemoteOperation { uploadFileOperation.execute(client) }
 
-        if (result == Unit && behavior == UploadBehavior.MOVE) {
-            removeLocalFile()
+        if (result == Unit) {
```

Comment:
There isn't a behaviour related with this result.

Reply:
Yes, here it is not necessary to handle this condition `behavior == UploadBehavior.MOVE` because the file we are going to delete is in the temporal folder. We this file should always be deleted because if not we will have a lot files in temporal folder.  So, to delete the file in the `sourcePath` (new parameter in database) will be done in the `RemoveSourceFileWorker`.